extensions/wikia/Email/i18n/Founder.i18n.php

```diff
@@ -45,6 +80,41 @@
+	'emailext-founder-3-days-extended-summary' => 'Additional line explaining the email contains tips for the founder to create a healthy and active wiki',
+	'emailext-founder-3-days-add-videos-header' => "Header for the link to add videos to the wiki",
+	'emailext-founder-3-days-add-videos-details' => "Line encouraging the user to add videos to the wiki",
+	'emailext-founder-3-days-update-theme-header' => "Header for the link to change themes, colors, and wordmarks on the wiki",
+	'emailext-founder-3-days-update-theme-details' => "Line encouraging the user to customize their wiki's themes, colors, and wordmarks using Wikia's Theme Designer",
+	'emailext-founder-3-days-wam-header' => "Header for the link to WAM which presents the most successful wikis on Wikia. Meant to provide inspiration for the founder and their wiki",
+	'emailext-founder-3-days-wam-details' => "Line encouraging the founder to visit WAM to see the successful wikis for inspiration",
+	'emailext-founder-3-days-need-help' => "Line which asks if the founder needs help and, if so, to visit their community.wikia.com wiki to get help from Wikia community support. http://community.wikia.com should point to language appropriate community wiki",
+	'emailext-founder-3-days-great-work' => "Line which enthusiastically tells the founder to keep doing good work",
+	'emailext-founder-10-days-subject' => "Subject for the email a founder receives 10 days after creating their wiki.",
+	'emailext-founder-10-days-summary' => "Intro line which reminds the founder it's been 10 days since they started their wiki",
+	'emailext-founder-10-days-extended-summary' => "Line telling the founder their wiki is looking good, and that now it's time to promote it to share it with the world",
+	'emailext-founder-10-days-sharing-header' => "Header for the section which encourages the founder to share their wiki on social media using the share buttons",
+	'emailext-founder-10-days-sharing-details' => "Line encouraging the founder to share their wiki on social media using the share buttons",
+	'emailext-founder-10-days-email-power-header' => "Header for the section encouraging the founder to email users to get them more involved and active on the wiki",
+	'emailext-founder-10-days-email-power-details' => "Line encouraging the founder to email users to get them more involved and active on the wiki",
+	'emailext-founder-10-days-email-google-header' => "Header link which gives the founder tips on improving their SEO. See 'emailext-founder-get-with-google'",
+	'emailext-founder-10-days-email-google-details' => "Line which reminds the user the internet is vast and they should work on improving their SEO to draw more traffic",
+	'emailext-founder-get-with-google' => "Link to page on community.wikia.com which gives tips on helping with SEO.",
+	'emailext-founder-10-days-email-what-next' => "Line which tells the user we're excited to see what they're going to do with their wiki",
```

Comment:
Thank you for recognizing that. It's been a long, hard road, but I think it's finally starting to pay off.